client/components/phone-input/index.jsx

```diff
@@ -16,21 +18,22 @@ import { countries } from './data';
 const PhoneInput = React.createClass( {
 	propTypes: {
 		onChange: React.PropTypes.func,
-		initialValue: React.PropTypes.string,
-		selectedCountryCode: React.PropTypes.string.isRequired,
+		value: React.PropTypes.string,
+		initialCountryCode: React.PropTypes.string,
 		countriesList: React.PropTypes.object.isRequired
 	},
 
 	getDefaultProps() {
 		return {
-			initialValue: '',
+			value: '',
 			onChange: noop,
+			initialCountryCode: 'us'
 		};
 	},
 
 	getInitialState() {
-		const selectedCountry = countries[ this.props.selectedCountryCode ];
-		const formattedNumber = formatNumber( this.props.initialValue, selectedCountry );
+		const selectedCountry = countries[ this.props.initialCountryCode ];
```

Comment:
This is a good idea if no number is passed. Otherwise, I'd say we should try to get the country from the number passed. Found this by reviewing #10313.